React/Base/macOS/RCTUIKit.m

```diff
@@ -423,6 +423,13 @@ - (void)setBackgroundColor:(NSColor *)backgroundColor
   }
 }
 
+// We purposely don't use RCTCursor for the parameter type here because it would introduce an import cycle:
+// RCTUIKit > RCTCursor > RCTConvert > RCTUIKit
+- (void)setCursor:(NSInteger)cursor
+{
+  // This method is required to be defined due to [RCTVirtualTextViewManager view] returning a RCTUIView.
+}
+
```

Comment:
If we implement this method, could any `RCTUIView` backed native component (Like... Text) also set a cursor?

Reply:
I implementing cursor on Text is more complicated because Text can nest Text and when it's nested it doesn't get a dedicated NSView and instead becomes attributed text. Internally we have cursors working on Text and nested Text, but it required changing Text to render NSTextView instead of something more like the iOS approach to Text rendering. We plan to discuss a few options related to this in our next sync.